db/seeds.rb

```diff
@@ -84,60 +84,60 @@
 sleep(0.2);
 
-o16 = Organization.create(name: 'Battlefly', address: '225 Smithe St #201, Vancouver, BC V6B 4X7, Canada', overview: '"Battlefy is the simplest way to start, manage, and find esports tournaments | Create. Compete. Conquer."', employee_count: 20, tech_team_size: 20, website: 'https://battlefy.com/', logo: 'https://pbs.twimg.com/profile_images/693214845928230912/ncCbhul7_400x400.png', twitter: 'battlefly', published: false)
+o16 = Organization.create(name: 'Battlefly', address: '225 Smithe St #201, Vancouver, BC V6B 4X7, Canada', overview: '"Battlefy is the simplest way to start, manage, and find esports tournaments | Create. Compete. Conquer."', employee_count: 20, tech_team_size: 20, website: 'https://battlefy.com/', logo: 'https://pbs.twimg.com/profile_images/693214845928230912/ncCbhul7_400x400.png', twitter: 'battlefly', published: true)
 sleep(0.2);
 
-o17 = Organization.create(name: 'Medeo', address: '208 - 128 West 6th Ave., Vancouver, BC V5Y 1K6, Canada', overview: '"Medeo is QHR\'s virtual care technology which connects health providers with their patients online for private video visits, secure messaging, and other virtual care applications. Medeo gives health providers the freedom to practice anywhere, any time. Patients are able to connect with their health providers via smartphone (iOS or Android), tablet, or computer."', employee_count: 5, tech_team_size: 5, website: 'https://medeo.ca/', logo: 'https://pbs.twimg.com/profile_images/560135744543412224/s36EIkQU_400x400.png', twitter: 'medeo', published: false)
+o17 = Organization.create(name: 'Medeo', address: '208 - 128 West 6th Ave., Vancouver, BC V5Y 1K6, Canada', overview: '"Medeo is QHR\'s virtual care technology which connects health providers with their patients online for private video visits, secure messaging, and other virtual care applications. Medeo gives health providers the freedom to practice anywhere, any time. Patients are able to connect with their health providers via smartphone (iOS or Android), tablet, or computer."', employee_count: 5, tech_team_size: 5, website: 'https://medeo.ca/', logo: 'https://pbs.twimg.com/profile_images/560135744543412224/s36EIkQU_400x400.png', twitter: 'medeo', published: true)
 sleep(0.2);
 
 o18 = Organization.create(name: 'Drive Digital', address: '404 - 1120 Hamilton St, Vancouver, BC V6B 2S2, Canada', overview: '"Drive Digital is a boutique web design agency specializing in web strategy, usability, creative, design and development of amazing websites and applications."', employee_count: 27, tech_team_size: 27, website: 'http://www.drivedigital.ca',logo: 'https://pbs.twimg.com/profile_images/378800000182289366/21f2345174df6f4f8886371cd39674db_400x400.png', twitter: 'drive_digital', published: false)
```

Comment:
Thank you for updating the seeds to have some show published for the maps to work. 👍